RentalCRUD.cs

```diff
@@ -0,0 +1,112 @@
+            allRentals = JsonConvert.DeserializeObject<List<Rental>>(rentalJsonString);
+
+            foreach (Rental rental in allRentals)
+            {
+                if (Convert.ToInt32(Id).Equals(rental.Id))
+                {
+                    Console.WriteLine($"Id: {rental.Id}");
+                    Console.WriteLine($"RentalDuration: {rental.RentalDuration}");
+                    Console.WriteLine($"RentalClientId: {rental.RentalClientId}");
+                    Console.WriteLine($"RentalCarId: {rental.RentalCarId}");
+                    Console.WriteLine($"RentalReturnDate: {rental.RentalReturnDate}");
+                    return rental;
+                }
+            }
+            return rental;
+        }
+
+        public Rental Create(Rental rental)
+        {
+            allRentals = JsonConvert.DeserializeObject<List<Rental>>(rentalJsonString);
+            Console.WriteLine("Enter the Id:");
+            rental.Id = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter the Rental duration:");
+            rental.RentalDuration = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter the Client Id:");
+            rental.RentalClientId = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter the Rental Car Id:");
+            rental.RentalCarId = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter the Rental Returned Date:");
+            rental.RentalReturnDate = Console.ReadLine();
+            allRentals.Add(rental);
+            Console.WriteLine("Rental was added succesfully !!");
+            string rentalJson = JsonConvert.SerializeObject(allRentals);
+            File.WriteAllText(rentalJsonFile, rentalJson);
+            return rental;
+        }
+        public void Delete(string Id)
+        {
+
+            allRentals = JsonConvert.DeserializeObject<List<Rental>>(rentalJsonString);
+
+            foreach (Rental element in allRentals)
+            {
+                if (Id.Equals(element.Id))
+                {
+                    Console.WriteLine($"Idt: {Id} deleted !!");
+                    allRentals.Remove(element);
+                    string rentaljson = JsonConvert.SerializeObject(allRentals);
+                    File.WriteAllText(rentalJsonFile, rentaljson);
+                    break;
+                }
+            }
+        }
+        public void Update(string Id)
+        {
+            allRentals = JsonConvert.DeserializeObject<List<Rental>>(rentalJsonString);
+            for (var i = 0; i < allRentals.Count; i++)
+            {
+                if (Convert.ToInt32(Id) == (allRentals[i].Id))
+                {
+                    Console.WriteLine("Enter the NEW id:");
+                    allRentals[i].Id = Convert.ToInt32(Console.ReadLine());
+                    Console.WriteLine("Enter the NEW Rental Duration:");
+                    allRentals[i].RentalDuration = Convert.ToInt32(Console.ReadLine());
```

Comment:
Asumiendo que el método recibe un parámetro `rental` del tipo `Rental`.
```suggestion
                    allRentals[i].RentalDuration = rental.RentalDuration;
```
Misma sugerencia para los otros valores. @csampedro si queres avisame y lo vemos en estos días 👍